plugin/evm/customrawdb/accessors_snapshot_ext.go

```diff
@@ -5,7 +5,7 @@ package customrawdb
 
 import (
 	"github.com/ava-labs/libevm/common"
-	"github.com/ava-labs/libevm/core/rawdb"
+	ethrawdb "github.com/ava-labs/libevm/core/rawdb"
```

Comment:
Why does this need to be aliased? We've completely deleted the local `./core/rawdb` so `libevm/core/rawdb` should just be added to the list of allowed imports.